src/esp/bindings/GfxBindings.cpp

```diff
@@ -98,9 +98,11 @@ void initGfxBindings(py::module& m) {
              self.renderEnter();
              return &self;
            })
-      .def("__exit__", [](RenderTarget& self, const py::object& exc_type,
-                          const py::object& exc_value,
-                          const py::object& traceback) { self.renderExit(); })
+      .def(
+          "__exit__",
+          [](RenderTarget& self, CORRADE_UNUSED const py::object& exc_type,
+             CORRADE_UNUSED const py::object& exc_value,
+             CORRADE_UNUSED const py::object& traceback) { self.renderExit(); })
```

Comment:
I'd just remove the names of the variables instead

```cpp
          [](RenderTarget& self, const py::object&,
             const py::object&,
             const py::object&) { self.renderExit(); })
```

that should make clang-tidy happy as well